dashboard/test/controllers/javabuilder_sessions_controller_test.rb

```diff
@@ -87,12 +87,35 @@ class JavabuilderSessionsControllerTest < ActionController::TestCase
     assert_equal "{\"useNeighborhood\":\"true\"}", decoded_token[0]['options']
   end
 
-  test 'student of authorized teacher can get access token' do
+  test 'student of authorized teacher without csa section cannot get access token' do
     teacher = create(:authorized_teacher)
     section = create(:section, user: teacher, login_type: 'word')
     student_1 = create(:follower, section: section).student_user
     sign_in(student_1)
     get :get_access_token, params: {channelId: @fake_channel_id, executionType: 'RUN', miniAppType: 'console'}
+    assert_response :forbidden
+    section.destroy
```

Comment:
I see we do this elsewhere, for my understanding -- why?

Reply:
I think it's to keep the test db clean but I'm thinking it may not be necessary